src/game.zig

```diff
@@ -765,6 +768,117 @@ pub fn hyperSpeedToggle() void {
 	Player.hyperSpeed.store(!Player.hyperSpeed.load(.monotonic), .monotonic);
 }
 
+const touchRetraction = @as(Vec3d, @splat(0.0001));
+const touchExpansion = 0.01;
+
+fn isBlockIntersecting(block: ?Block, posX: i32, posY: i32, posZ: i32, center: Vec3d, extent: Vec3d) bool {
```

Comment:
Don't use an optional parameter, if you assert in the next line (by using `.?`) that it cannot be null.